app/views/owned_tag_sets/show_options.html.erb

```diff
@@ -1,14 +1,13 @@
 <% cache("tagsets_#{@tag_set_ids.join('-')}_#{@tag_sets.collect(&:updated_at).sort.last}", skip_digest: true) do %>
   <h2>
-   <%= ts("#{@tag_type.humanize.titleize} Options") %>
+   <%= t("tag_types.#{@tag_type}").humanize.titleize %> Options
```

Comment:
This is in a cache section so shoudn't the locale be part of the cache key ?

Reply:
According to @bingeling:
>It's not needed in this case, but once we have a localized UI the locale does need to be part of the cache key. It shouldn't cause any future problems if we don't have it in the cache key now, Rails will generate a new cache fragment whenever we make this switch.
So it wouldn't hurt to add the locale to the key, but not having it there shouldn't keep us from merging this.